### PR TITLE
[DNM] travis: Add utility to check missing porting config changes

### PR DESCRIPTION
**Pull request by michal-narajowski**
Use porting/update_generated_files.sh script to check if
we need to update config files in porting before merging a PR.

### CODE REVIEW

File: nimble/host/syscfg.yml [hunk 1]
```diff
@@ -121,7 +121,7 @@ syscfg.defs:
     # Security manager settings.
     BLE_SM_LEGACY:
         description: 'Security manager legacy pairing.'
-        value: 1
+        value: 0
```
Comment (rymanluk) [inline nimble/host/syscfg.yml:124]: this seems to not be part of this patch

Comment (michal-narajowski) [reply nimble/host/syscfg.yml:124]: Yes, this PR is just for testing
